Tribler/Core/CacheDB/SqliteCacheDBHandler.py

```diff
@@ -1078,31 +1077,8 @@ def freeSpace(self, torrents2del):
         # sql_insert =  "insert into Torrent (torrent_id, infohash, relevance) values (?,?,?)"
         # self._db.executemany(sql_insert, torrent_id_infohashes)
 
-        torrent_dir = self.session.get_torrent_collecting_dir()
         deleted = 0  # deleted any file?
         insert_files = []
```

Comment:
This `freeSpace()` used to delete torrent files in the torrent collecting directory to free space. Since we have migrated to `leveldb`, those code doesn't work any more. So no, it's not used. Later, we also need to fix this `freeSpace()` or simply remove it if we don't need it any more.